libkpod/rename.go

```diff
@@ -0,0 +1,108 @@
+package libkpod
+
+import (
+	"encoding/json"
+	"path/filepath"
+
+	"github.com/docker/docker/pkg/ioutils"
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/opencontainers/runtime-tools/generate"
+)
+
+const (
+	configFile        = "config.json"
+	nameAnnotation    = "io.kubernetes.cri-o.Name"
+	metadatAnnotation = "io.kubernetes.cri-o.Metadata"
```

Comment:
Use values from `github.com/kubernetes-incubator/cri-o/pkg/annotations` instead of adding copies here.